frontend/src/components/canvas/FlowCanvas.tsx

```diff
@@ -92,8 +92,8 @@ const useNodeTypes = ({ nodeTypesConfig }: { nodeTypesConfig: NodeTypesConfig |
           acc[node.name] = InputNode;
         } else if (node.name === 'RouterNode') {
           acc[node.name] = RouterNode;
-        } else if (node.name === 'MergeNode') {
-          acc[node.name] = (props: any) => <MergeNode {...props} />;
+        } else if (node.name === 'CoalesceNode') {
+          acc[node.name] = CoalesceNode;
```

Comment:
@preet-bhadra can you make sure there are no linter errors here? If there are then let's use the functional style as it was done for merge node.